src/networks.js

```diff
@@ -0,0 +1,55 @@
+import env from './environment'
+
+const DEFAULT_IPFS_GATEWAY = 'https://ipfs.eth.aragon.network/ipfs'
+const DEFAULT_ORG_LOCATION = 'agreement3.aragonid.eth'
+
+const NETWORK_ENVIRONMENTS = new Map([
+  [
+    'ethereum',
+    {
+      chainId: 1,
+      legacyNetworkType: 'main',
```

Comment:
Just wondering, when do we still need this (we should be eliminating it as much as possible 🔪)?

Reply:
See https://github.com/aragon/network-dashboard/pull/8#discussion_r466980726